### PR TITLE
Add 7.07.2016

### PR DESCRIPTION
**Pull request by PavelMihayeu**

### CODE REVIEW

File: Service/InvalidLoginService.robot [hunk 1]
```diff
@@ -0,0 +1,16 @@
+*** Settings ***
+Resource          ../Page/LoginPage.robot
+Resource          ../Page/NavigatorPage.robot
+
+*** Keywords ***
+Login With Invalid User Name And Wait Error Message UI
+    Login
```
Comment (PavelMihayeu) [inline Service/InvalidLoginService.robot]: remove this service

File: test/invalid_login.robot [hunk 2]
```diff
@@ -0,0 +1,12 @@
+*** Settings ***
+Resource          ../Service/InvalidLoginService.robot
+
+*** Test Cases ***
+1. Invalid Username
+    Login With Invalid User Name And Wait Error Message UI
```
Comment (PavelMihayeu) [inline test/invalid_login.robot]: invalidLogin | ${ADMIN_PASSWORD}

File: test/invalid_login.robot [hunk 3]
```diff
@@ -0,0 +1,12 @@
+*** Settings ***
+Resource          ../Service/InvalidLoginService.robot
```
Comment (PavelMihayeu) [inline test/invalid_login.robot]: Resource          ../Service/LoginService.robot

File: test/invalid_login.robot [hunk 4]
```diff
@@ -0,0 +1,12 @@
+*** Settings ***
```
Comment (PavelMihayeu) [inline test/invalid_login.robot:1]: [Test Template]    Login And Check Error Message Template

File: test/invalid_login.robot [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+Resource          ../Service/InvalidLoginService.robot
+
+*** Test Cases ***
+1. Invalid Username
+    Login With Invalid User Name And Wait Error Message UI
+
+2. Invalid Password
+    Login With Invalid Password And Wait Error Message UI
```
Comment (PavelMihayeu) [inline test/invalid_login.robot]: ${ADMIN_USER} | invalidPassword

File: test/invalid_login.robot [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+3. Empty Password
+    Login With Empty Password And Wait Error Message UI
```
Comment (PavelMihayeu) [inline test/invalid_login.robot]: ${ADMIN_USER} | ${EMPTY}

File: Page/NavigatorPage.robot [hunk 7]
```diff
@@ -1,7 +1,13 @@
 *** Variables ***
 ${TOP_LOGO_IMAGE_LOCATOR}    ${EMPTY}
+${LOGIN_ERROR_MESSAGE_LOCATOR}    ${EMPTY}
```
Comment (pavelpantseleyeu) [inline Page/NavigatorPage.robot:3]: These locators are located on login page. Move them in login page

**Pull request closed without merging**